roles/common/tasks/pod_tests.yml

```diff
@@ -0,0 +1,29 @@
+---
+- block:
+    - name: Get Pod Instance "{{ common_pod_status_str }}"
+      ansible.builtin.shell:
+        cmd: |
+          oc get pods -n "{{ common_pod_nspace }}" | grep "{{ item }}" | grep "{{ common_pod_status_str }}" | awk '{print $1;}'
+      register: podinstance
+      failed_when:
+        - podinstance.stdout_lines | length == 0
+      changed_when: false
+
+    - name: Check terminated pod {{ common_pod_test_id }}
```

Comment:
@ayefimov-1 What is the purpose of this second task? It looks like it is repeating the previous test
It might also be mis-named?
Is this just supposed to be getting more information on the pod by name?

If it is there for more information, then I think we can make it optional to run (only run of the pod exists, and don't let this failures effect the test run) since it is informational and not a test

Reply:
The goal of this test is to validate the pods existence and status. 

Line 12 is misnamed and should be called "Check Pod Status {{ common_pod_test_id }}".

Reply:
The first tasks is filtering on status.
If the second take is trying to check status, then the first task should not filter on the status.

To leave the first tasks as-is renders the second unnecessary.